### DISCUSSION TITLE
Show resources thumbnails in the quick load menu

### DISCUSSION THREAD

**Discussion by mieldepoche** [godotengine/godot-proposals#7455] [Editor]
This is what the `quick load` menu (in the inspector, RMB on a resource field > `quick load`) shows when looking for a texture:

![image](https://github.com/godotengine/godot-proposals/assets/71937306/e294ef75-f6df-42cb-8573-d48ed61b5df8)

it's the same for all resources: only the resource type icon is shown, no thumbnail.
This forces you to parse a long list of files.

The solution is clear: show resource thumbnails. But how?

1. show the thumbnail instead of the icon, using a bigger image size, keeping the current list view (so, thicker items) ?
2. show a grid view of the thumbnail and file names? (con: can't show full path)
3. allow toggling between those two options?
4. ?

**Comment by timothyqiu** [comment]
See also #3788
